cd  /data/NCATS_ifx/data/mRNASeq/Vukasin       && mkdir -p bam/Sample_VJ3736_1_S16       && STAR         --runThreadN $SLURM_CPUS_PER_TASK         --genomeDir /fdb/STAR_current/GENCODE/Gencode_human/release_27/genes-75         --sjdbOverhang 75 \ --outSAMunmapped Within –-outFilterType BySJout         --outFilterMultimapNmax 20 --outFilterMismatchNmax 999         --outFilterMismatchNoverLmax 0.04 --alignIntronMin 20         --alignIntronMax 1000000 --alignMatesGapMax 1000000         --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 -- sjdbScore 1         --readFilesIn /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_1_S16_1P.fastq.gz /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_1_S16_2P.fastq.gz         --readFilesCommand zcat         --outSAMtype BAM SortedByCoordinate         --outFileNamePrefix bam/Sample_VJ3736_1_S16/_hg38


cd  /data/NCATS_ifx/data/mRNASeq/Vukasin       && mkdir -p bam/Sample_VJ3736_2_S17       && STAR         --runThreadN $SLURM_CPUS_PER_TASK         --genomeDir /fdb/STAR_current/GENCODE/Gencode_human/release_27/genes-75         --sjdbOverhang 75 \ --outSAMunmapped Within –-outFilterType BySJout         --outFilterMultimapNmax 20 --outFilterMismatchNmax 999         --outFilterMismatchNoverLmax 0.04 --alignIntronMin 20         --alignIntronMax 1000000 --alignMatesGapMax 1000000         --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 -- sjdbScore 1         --readFilesIn /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_2_S17_1P.fastq.gz /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_2_S17_2P.fastq.gz         --readFilesCommand zcat         --outSAMtype BAM SortedByCoordinate         --outFileNamePrefix bam/Sample_VJ3736_2_S17/_hg38


cd  /data/NCATS_ifx/data/mRNASeq/Vukasin       && mkdir -p bam/Sample_VJ3736_3_S18       && STAR         --runThreadN $SLURM_CPUS_PER_TASK         --genomeDir /fdb/STAR_current/GENCODE/Gencode_human/release_27/genes-75         --sjdbOverhang 75 \ --outSAMunmapped Within –-outFilterType BySJout         --outFilterMultimapNmax 20 --outFilterMismatchNmax 999         --outFilterMismatchNoverLmax 0.04 --alignIntronMin 20         --alignIntronMax 1000000 --alignMatesGapMax 1000000         --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 -- sjdbScore 1         --readFilesIn /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_3_S18_1P.fastq.gz /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_3_S18_2P.fastq.gz         --readFilesCommand zcat         --outSAMtype BAM SortedByCoordinate         --outFileNamePrefix bam/Sample_VJ3736_3_S18/_hg38


cd  /data/NCATS_ifx/data/mRNASeq/Vukasin       && mkdir -p bam/Sample_VJ3736_4_S19       && STAR         --runThreadN $SLURM_CPUS_PER_TASK         --genomeDir /fdb/STAR_current/GENCODE/Gencode_human/release_27/genes-75         --sjdbOverhang 75 \ --outSAMunmapped Within –-outFilterType BySJout         --outFilterMultimapNmax 20 --outFilterMismatchNmax 999         --outFilterMismatchNoverLmax 0.04 --alignIntronMin 20         --alignIntronMax 1000000 --alignMatesGapMax 1000000         --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 -- sjdbScore 1         --readFilesIn /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_4_S19_1P.fastq.gz /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_4_S19_2P.fastq.gz         --readFilesCommand zcat         --outSAMtype BAM SortedByCoordinate         --outFileNamePrefix bam/Sample_VJ3736_4_S19/_hg38


cd  /data/NCATS_ifx/data/mRNASeq/Vukasin       && mkdir -p bam/Sample_VJ3736_5_S20       && STAR         --runThreadN $SLURM_CPUS_PER_TASK         --genomeDir /fdb/STAR_current/GENCODE/Gencode_human/release_27/genes-75         --sjdbOverhang 75 \ --outSAMunmapped Within –-outFilterType BySJout         --outFilterMultimapNmax 20 --outFilterMismatchNmax 999         --outFilterMismatchNoverLmax 0.04 --alignIntronMin 20         --alignIntronMax 1000000 --alignMatesGapMax 1000000         --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 -- sjdbScore 1         --readFilesIn /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_5_S20_1P.fastq.gz /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_5_S20_2P.fastq.gz         --readFilesCommand zcat         --outSAMtype BAM SortedByCoordinate         --outFileNamePrefix bam/Sample_VJ3736_5_S20/_hg38


cd  /data/NCATS_ifx/data/mRNASeq/Vukasin       && mkdir -p bam/Sample_VJ3736_6_S21       && STAR         --runThreadN $SLURM_CPUS_PER_TASK         --genomeDir /fdb/STAR_current/GENCODE/Gencode_human/release_27/genes-75         --sjdbOverhang 75 \ --outSAMunmapped Within –-outFilterType BySJout         --outFilterMultimapNmax 20 --outFilterMismatchNmax 999         --outFilterMismatchNoverLmax 0.04 --alignIntronMin 20         --alignIntronMax 1000000 --alignMatesGapMax 1000000         --alignSJoverhangMin 8 --alignSJDBoverhangMin 1 -- sjdbScore 1         --readFilesIn /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_6_S21_1P.fastq.gz /data/NCATS_ifx/data/mRNASeq/Vukasin/VJ3736_6_S21_2P.fastq.gz         --readFilesCommand zcat         --outSAMtype BAM SortedByCoordinate         --outFileNamePrefix bam/Sample_VJ3736_6_S21/_hg38


